src/thx/Ord.hx

```diff
@@ -2,15 +2,34 @@ package thx;
 
 import thx.Semigroup;
 
-enum Ordering {
+abstract Ordering(OrderingImpl) from OrderingImpl to OrderingImpl {
+  @:from public static function fromInt(value : Int) : Ordering
```

Comment:
These implicit coercions make me nervous. If the function is inlined, isn't it just as good to make an explicit .fromInt or .toInt call?